sdk/textanalytics/azure-ai-textanalytics/samples/async_samples/sample_analyze_sentiment_with_opinion_mining_async.py

```diff
@@ -94,67 +61,75 @@ async def analyze_sentiment_with_opinion_mining(self):
-        positive_mined_opinions = []
-        mixed_mined_opinions = []
-        negative_mined_opinions = []
+        print(
+            "\nIn order to do that, I'm going to extract aspects that have a negative sentiment. "
+            "I'm going to map aspect to the mined opinion object we get back to aggregate the reviews by aspect. "
+        )
+        aspect_to_complaints = {}
 
         for document in doc_result:
             for sentence in document.sentences:
                 for mined_opinion in sentence.mined_opinions:
                     aspect = mined_opinion.aspect
-                    if aspect.sentiment == "positive":
-                        positive_mined_opinions.append(mined_opinion)
-                    elif aspect.sentiment == "mixed":
-                        mixed_mined_opinions.append(mined_opinion)
-                    else:
-                        negative_mined_opinions.append(mined_opinion)
-
-        print("\n\nLet's look at the {} positive opinions users have expressed for aspects of this hotel".format(len(positive_mined_opinions)))
-        for mined_opinion in positive_mined_opinions:
-            print("...Reviewers have the following opinions for the overall positive '{}' aspect of the hotel".format(mined_opinion.aspect.text))
-            for opinion in mined_opinion.opinions:
-                print("......'{}' opinion '{}'".format(opinion.sentiment, opinion.text))
-
-        print("\n\nNow let's look at the {} mixed opinions users have expressed for aspects of this hotel".format(len(mixed_mined_opinions)))
-        for mined_opinion in mixed_mined_opinions:
-            print("...Reviewers have the following opinions for the overall mixed '{}' aspect of the hotel".format(mined_opinion.aspect.text))
-            for opinion in mined_opinion.opinions:
-                print("......'{}' opinion '{}'".format(opinion.sentiment, opinion.text))
-
-        print("\n\nFinally, let's see the {} negative opinions users have expressed for aspects of this hotel".format(len(negative_mined_opinions)))
-        for mined_opinion in negative_mined_opinions:
-            print("...Reviewers have the following opinions for the overall negative '{}' aspect of the hotel".format(mined_opinion.aspect.text))
-            for opinion in mined_opinion.opinions:
-                print("......'{}' opinion '{}'".format(opinion.sentiment, opinion.text))
+                    if aspect.sentiment == 'negative':
+                        aspect_to_complaints.setdefault(aspect.text, [])
+                        aspect_to_complaints[aspect.text].append(mined_opinion)
+
+        print("\nLet's now go through the aspects of our hotel people have complained about and see what users have specifically said")
+
+        for aspect, complaints in aspect_to_complaints.items():
+            print("Users have made {} complaints about '{}', specifically saying that it's '{}'".format(
```

Comment:
nit since it evaluates to 1 for some

```suggestion
            print("Users have made {} complaint(s) about '{}', specifically saying that it's '{}'".format(
```